sklearn/cross_validation.py

```diff
@@ -32,6 +32,14 @@
 from .metrics.scorer import check_scoring
 from .utils.fixes import bincount
 
+
+warnings.warn("This module has been deprecated in favor of the "
+              "model_selection module into which all the refactored classes "
+              "and functions are moved. Also note that the interface of the "
+              "new CV iterators are different from that of this module. "
+              "Refer model_selection.split for more info.", DeprecationWarning)
```

Comment:
it's "Refer to" and I would not reference ``split`` as I still think it should be private (pending comments from the others).